packages/react-devtools-shared/src/__tests__/__snapshots__/profilingCache-test.js.snap

```diff
@@ -3207,6 +3244,2081 @@ Object {
+      "snapshots": Array [],
+    },
+  ],
+  "version": 5,
+}
+`;
+
+exports[`ProfilingCache should properly detect when a component is missing a shallow/deep memo for props: CommitDetails commitIndex: 0 1`] = `
+Object {
+  "changeDescriptions": null,
+  "duration": 0,
+  "effectDuration": null,
+  "fiberActualDurations": Map {
+    1 => 0,
+    2 => 0,
+    3 => 0,
+    4 => 0,
+    5 => 0,
+    6 => 0,
+  },
+  "fiberSelfDurations": Map {
+    1 => 0,
+    2 => 0,
+    3 => 0,
+    4 => 0,
+    5 => 0,
+    6 => 0,
+  },
+  "passiveEffectDuration": null,
+  "perfInsights": Map {
+    2 => Object {
+      "arePropsDeepEqual": false,
+      "arePropsEqual": false,
+      "didHooksChange": false,
+      "isContextDeepEqual": false,
+      "isContextEqual": false,
+      "isFirstMount": true,
+      "isStateDeepEqual": false,
+      "isStateEqual": false,
+      "nonMemoizedProps": null,
+    },
+    3 => Object {
+      "arePropsDeepEqual": false,
+      "arePropsEqual": false,
+      "didHooksChange": false,
+      "isContextDeepEqual": false,
+      "isContextEqual": false,
+      "isFirstMount": true,
+      "isStateDeepEqual": false,
+      "isStateEqual": false,
+      "nonMemoizedProps": null,
+    },
+    4 => Object {
+      "arePropsDeepEqual": false,
+      "arePropsEqual": false,
+      "didHooksChange": false,
+      "isContextDeepEqual": false,
+      "isContextEqual": false,
+      "isFirstMount": true,
+      "isStateDeepEqual": false,
+      "isStateEqual": false,
+      "nonMemoizedProps": null,
+    },
+    5 => Object {
+      "arePropsDeepEqual": false,
+      "arePropsEqual": false,
+      "didHooksChange": false,
+      "isContextDeepEqual": false,
+      "isContextEqual": false,
+      "isFirstMount": true,
+      "isStateDeepEqual": false,
+      "isStateEqual": false,
+      "nonMemoizedProps": null,
+    },
+    6 => Object {
+      "arePropsDeepEqual": false,
+      "arePropsEqual": false,
+      "didHooksChange": false,
+      "isContextDeepEqual": false,
+      "isContextEqual": false,
+      "isFirstMount": true,
+      "isStateDeepEqual": false,
+      "isStateEqual": false,
+      "nonMemoizedProps": null,
+    },
+  },
+  "priorityLevel": "Immediate",
+  "timestamp": 0,
+  "updaters": Array [
+    Object {
+      "displayName": "Anonymous",
+      "hocDisplayNames": null,
+      "id": 1,
+      "key": null,
+      "type": 11,
+    },
+  ],
+}
+`;
+
+exports[`ProfilingCache should properly detect when a component is missing a shallow/deep memo for props: CommitDetails commitIndex: 1 1`] = `
+Object {
+  "changeDescriptions": null,
+  "duration": 0,
+  "effectDuration": null,
+  "fiberActualDurations": Map {
+    2 => 0,
+    1 => 0,
+  },
+  "fiberSelfDurations": Map {
+    2 => 0,
+    1 => 0,
+  },
+  "passiveEffectDuration": null,
+  "perfInsights": Map {
+    2 => Object {
+      "arePropsDeepEqual": true,
+      "arePropsEqual": true,
+      "didHooksChange": false,
+      "isContextDeepEqual": false,
```

Comment:
Why is this false? Seems like it should be true. There isn't even context for one. But also the only case where `isContextDeepEqual === false && isContextEqual === true` would be if a context object were _mutated_ (which shouldn't happen)?

Reply:
`isContextDeepEqual` is hardcoded to `false` and is on the TODO list :)

Reply:
Hmm. I see. Somewhat uncomfortable with landing that in a broken state, though I guess it matters less if so long as the feature flag is disabled.

Reply:
I wanted to get it in a good working state. I'll add it as a separate PR so its easier to review